README.md

```diff
@@ -8,3 +8,71 @@ For further information about the methods contained in this module see the [meth
 
 For user documentation and example data relating to the methods in this module see the [supporting information](https://github.com/ONSdigital/sml-supporting-info)
 
+### Automated testing
+In order to ensure code quality, there is a manual test script provided __run_py_tools.sh__ which will run linting, code formatting checks, and the pytest suite. 
+
+It is often easy to forget to check code formatting before pushing to the remote repository, so there is the option of running the testing script automatically by using the git hook __pre-push__. This means that when __git push__ is run, the test script will be run first, and will abort the push if any of the tests fail.
+
+Git hooks cannot be pushed to the remote repository so if you would like this script to be run automatically you will need to follow these steps:
```

Comment:
just wondering why cant we push these into the repo?

Reply:
Git won't allow you to push or pull git hooks to/from the remote repository for security reasons